spring-integration-redis/src/test/java/org/springframework/integration/redis/util/RedisLockRegistryTests.java

```diff
@@ -341,21 +341,24 @@ public void testTwoThreadsWrongOneUnlocks() throws Exception {
 	@Test
 	@RedisAvailable
 	public void testExpireTwoRegistries() throws Exception {
-		RedisLockRegistry registry1 = new RedisLockRegistry(this.getConnectionFactoryForTest(), this.registryKey, 100);
-		RedisLockRegistry registry2 = new RedisLockRegistry(this.getConnectionFactoryForTest(), this.registryKey, 100);
+		RedisLockRegistry registry1 = new RedisLockRegistry(getConnectionFactoryForTest(), this.registryKey, 100);
+		RedisLockRegistry registry2 = new RedisLockRegistry(getConnectionFactoryForTest(), this.registryKey, 100);
 		Lock lock1 = registry1.obtain("foo");
 		Lock lock2 = registry2.obtain("foo");
 		assertTrue(lock1.tryLock());
 		assertFalse(lock2.tryLock());
+		this.thrown.expect(IllegalStateException.class);
+		this.thrown.expectMessage("Lock was released in the store due to expiration.");
 		waitForExpire("foo");
+		lock1.unlock();
 		assertTrue(lock2.tryLock());
 		assertFalse(lock1.tryLock());
```

Comment:
These two are redundant now - shouldn't the test for the new logic be in a different test?